msbuild/Xamarin.iOS.Tasks.Core/Tasks/MTouchTaskBase.cs

```diff
@@ -391,7 +391,7 @@ protected override string GenerateCommandLineCommands ()
 				args.Add ("--extension");
 
 			if (Debug) {
-				if (FastDev && IPhoneSdks.MonoTouch.SupportsFastDev)
+				if (FastDev)
```

Comment:
It's required due to the bump of Xamarin.MacDev which removed the SupportsX properties (as they aren't needed, all of them are for versions older than 10.0).